pyproject.toml

```diff
@@ -4,6 +4,7 @@ version = "2023.05.04"
 description = "Sample Programs in Every Programming Language"
 readme = "README.md"
 license = "MIT"
+package-mode = false
```

Comment:
This is good to know since I use Poetry with every project, even if it's a little script that I'm never going to package.